VideoAPI/Video.API/Mappings/EventRequestToEventCommandMapper.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using VideoApi.Contract.Requests;
+using VideoApi.DAL.Commands;
+using VideoApi.Domain.Enums;
+using VideoApi.Events.Models;
+
+namespace Video.API.Mappings
+{
+    public static class EventRequestMapper
+    {
+        public static SaveEventCommand MapEventRequestToEventCommand(Guid conferenceId, ConferenceEventRequest request)
+        {
+            GetRoomTypeEnums(request, out var transferTo, out var transferFrom);
+
+            var command = new SaveEventCommand(conferenceId, request.EventId, request.EventType,
+                request.TimeStampUtc, transferFrom, transferTo, request.Reason, request.Phone);
+            if (Guid.TryParse(request.ParticipantId, out var participantId))
+            {
+                command.ParticipantId = participantId;
+            }
+
+            command.TransferredFromRoomLabel = request.TransferFrom;
+            command.TransferredToRoomLabel = request.TransferTo;
+
+            return command;
+        }
+
+        public static CallbackEvent MapEventRequestToEventHandlerDto(Guid conferenceId, Guid participantId,
```

Comment:
did we need a null or empty check for these?

Reply:
were doing try parse in controller and not using result

Reply:
Conference Id and PartipantId is used for the eventhandler mapper whereas it's not mandatory for the event command